hs/Server.hs

```diff
@@ -58,6 +63,9 @@ runServer = do
               dir "depts" $ liftIO deptList,
               dir "timesearch" searchResponse,
               dir "calendar" $ lookCookieValue "selected-lectures" >>= calendarResponse,
+              -- dir "draw-json?gid=num" $ look "gid" >>= drawJsonResponse,
```

Comment:
They certainly can be - although you generally should mention that in the description so that I don't think you want it to be merged in (which is another purpose of pull requests, of course).